### PR TITLE
DHP-1072: Add JSON to table row to exporter3

### PR DESCRIPTION
**Pull request by DwayneJengSage**
See https://github.com/Sage-Bionetworks/BridgeWorkerPlatform/pull/180

Replaces https://github.com/Sage-Bionetworks/BridgeWorkerIntegrationTests/pull/61

### CODE REVIEW

File: src/test/java/org/sagebionetworks/bridge/exporter/integration/Exporter3Test.java [hunk 1]
```diff
@@ -612,7 +745,23 @@ public void uploadWithScheduleContext() throws Exception {
...
+        assessmentConfig.setConfig(ASSESSMENT_CONFIG);
+        assessmentsApi.updateAssessmentConfig(assessment.getGuid(), assessmentConfig).execute();
+
+        // Get study.
+        Study study = studiesApi.getStudy(STUDY_ID).execute().body();
+
+        // Make upload. This is a zip file with two files in it, metadata.json and assessmentResult.json.
+        byte[] zipFileContent = zip(SURVEY_UPLOAD_CONTENT_BY_FILENAME);
+
+        // todo the zip file seems to be malformed.
```
Comment (nategbrown9) [inline src/test/java/org/sagebionetworks/bridge/exporter/integration/Exporter3Test.java]: Is this todo still relevant?

**Review by nategbrown9 — APPROVED** [review]
Looks good.

File: src/test/java/org/sagebionetworks/bridge/exporter/integration/Exporter3Test.java (hunk 1 above)
Comment (DwayneJengSage) [reply src/test/java/org/sagebionetworks/bridge/exporter/integration/Exporter3Test.java]: Good catch. I left the TODO at the end of one coding session to remind myself of where I left off. Then for the next coding session, I immediately forgot about the TODO, but then went and solved the problem anyway. I guess the TODO didn't do it's job.

**Review by nategbrown9 — APPROVED** [review]
Looks good.

**Pull request merged**